server_admin/topics/authentication/conditions.adoc

```diff
@@ -41,3 +41,46 @@ Expected value in the attribute.
 Negate output:::
 You can negate the output.
 In other words, the attribute should not be present.
+
+==== Explicitly deny/allow access in conditional flows
+
+It's possible to explicitly either allow or deny access to the resources in a conditional flow.
+It's really suitable in conditional flows, where you can associate that with available conditions.
+There are provided two explicit authenticators `DenyAccess` and `AllowAccess`, which should solve the issue with access to the resources by conditions.
```

Comment:
try to avoid "there is" and "it is" which makes no sense grammatically. For example what is the subject in a "there is" sentence.
I suggest:  "The two authenticators `DenyAccess` and `AllowAccess` control access to the resources by conditions."